src/actions/guides/command_line_tasks/custom_tasks.cr

```diff
@@ -7,26 +7,47 @@ class Guides::CommandLineTasks::CustomTasks < GuideAction
 
   def markdown
     <<-MD
-    ## Custom Tasks
+    ## Creating Custom Tasks
 
-    Custom tasks you need will be placed in the `tasks` folder of your application. The three main things to creating a custom task is that your class inherits from `LuckyCli::Task`, it implements a `call` method, and includes a `banner`. Be sure to `require "lucky_cli"` at the top of your new task.
-
-    Lucky will infer the name of the task by using the name of your class. This includes using namespaces. (e.g. `Db::Migrate` becomes `lucky db.migrate`).
+    Custom tasks you need will be placed in the `tasks` folder of your application.
+    The three main things to creating a custom task is
+    1. Your class must inherit from `LuckyCli::Task`
+    2. It must implement a `call` method
+    3. It must include a `summary`
 
     ```crystal
     # tasks/generate_sitemaps.cr
-    require "lucky_cli"
-
     class GenerateSitemaps < LuckCli::Task
-      banner "Generate the sitemap.xml for this site"
+      summary "Generate the sitemap.xml for this site"
 
       def call
         # Implement your task here
       end
     end
     ```
 
+    Lucky will infer the name of the task by using the name of your class. This includes using namespaces. (e.g. `Db::Migrate` becomes `lucky db.migrate`, and `GenerateSitemaps` becomes `lucky generate_sitemaps`).
+
+    Optionally, if you want to customize the name of your task, you can use the `name` macro.
+
+    ```crystal
+    class GenerateSitemaps < LuckyCli::Task
+      summary "Generate the sitemap.xml for this site"
+      name "generate.sitemaps"
```

Comment:
Maybe something obviously different so it’s easier to reference later. “custom.task” maybe?